src/redecomp/transfer/MatrixTransfer.cpp

```diff
@@ -147,10 +150,21 @@ std::unique_ptr<mfem::HypreParMatrix> MatrixTransfer::ConvertToHypreParMatrix( m
   SLIC_ERROR_IF( sparse.Width() != parent_trial_fes_.GlobalVSize(),
                  "Width of sparse must match number of trial ParFiniteElementSpace global dofs." );
 
+  // TODO: mfem::SparseMatrix uses int for global column values; this needs to be HYPRE_BigInt to be compatible with
```

Comment:
Same comment re elevating this to an issue?

Reply:
Add name to comment